dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/publish/servers/FTFallback/server.xml

```diff
@@ -11,15 +11,17 @@
 <server>
 
     <include location="../fatTestPorts.xml"/>
-    
+
 	<featureManager>
-	    <feature>componenttest-1.0</feature>
+		<feature>componenttest-1.0</feature>
 		<feature>osgiconsole-1.0</feature>
 		<feature>localConnector-1.0</feature>
 		<feature>appSecurity-3.0</feature>
 		<feature>cdi-2.0</feature>
 		<feature>servlet-4.0</feature>
 		<feature>mpFaultTolerance-2.0</feature>
+		<feature>ejblite-3.2</feature>
+		<feature>appSecurity-3.0</feature>
```

Comment:
I wonder if a separate server would be better rather than introducing ejb to an existing one?

Reply:
Mmm, possibly, though that would also mean separating out the tests into different classes.

I'm happy to do it, I didn't bother to in the first instance because I'm lazy.